app/graphql/types/chapter.rb

```diff
@@ -0,0 +1,14 @@
+class Types::Chapter < Types::BaseObject
+  implements Types::MediaUnitInterface
+  description 'A chapter of a Manga'
+
+  field :id, ID, null: false
+
+  field :volume_number, Integer,
+    null: false,
+    description: 'The volume this chapter is related to'
+
+  field :published, GraphQL::Types::ISO8601DateTime,
```

Comment:
Could you change this to `released_at`?  You can pass `method: :published` to map it to the published field in the db